storage/debugbar/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
-!.gitignore
```

Comment:
This shouldn't be deleted (since we don't want to accidentally commit any files generated by the Laravel DebugBar package that are placed in here).

Reply:
@DFurnes oops! Seems like this got removed when I did the `git rm -rf --cached .` and then `git add .`. When I tried to re-add this, I got the message "If you decide to go ahead and use a name which begins with a dot the file will be hidden." Do you know how I can re-add this?

Reply:
Are you trying to add the `.gitignore` file in the Finder or macOS save dialog? I think you might have to make dotfiles from the command line.

Reply:
ooohhh I was trying in Sublime @DFurnes !